src/main/java/mate/academy/internet/shop/model/Order.java

```diff
@@ -0,0 +1,60 @@
+package mate.academy.internet.shop.model;
+
+import java.util.List;
+
+public class Order {
+    private Long id;
+    private User user;
+    private List<Item> items;
+    private Bucket bucket;
```

Comment:
Bucket is not needed at Order. please, remove this field

Reply:
Have a question